server/cla.go

```diff
@@ -103,21 +103,25 @@ func (s *Server) checkCLA(ctx context.Context, pr *model.PullRequest) {
 	_ = s.createRepoStatus(ctx, pr, status)
 }
 
-func (s *Server) getCSV() ([]byte, error) {
-	resp, err := http.Get(s.Config.SignedCLAURL)
+func (s *Server) getCSV(ctx context.Context) ([]byte, error) {
+	req, err := http.NewRequestWithContext(ctx, http.MethodGet, s.Config.SignedCLAURL, http.NoBody)
+	if err != nil {
+		return nil, err
+	}
+	r, err := http.DefaultClient.Do(req)
 	if err != nil {
 		mlog.Error("Unable to get CLA list", mlog.Err(err))
```

Comment:
Unrelated to this PR, but this is an anti-pattern. Logging and then returning the same error. We need to do a sweep to fix all of these.